packages/jellyfish-api-core/src/category/mining.ts

```diff
@@ -86,3 +96,11 @@ export interface MasternodeInfo {
   mintedblocks?: number
   lastblockcreationattempt?: string
 }
+
+export interface SmartFeeEstimation {
+  feerate?: number
+  errors?: string[]
+  blocks: number
+}
+
+type EstimateMode = 'UNSET' | 'ECONOMICAL' | 'CONSERVATIVE'
```

Comment:
Currently, everyone put it above, we can follow that also, should use enum also. Enum we can add docs. plain types we can't

You can follow this but have it for yourself: 

https://github.com/DeFiCh/jellyfish/blob/ece0fdc6e24e2f9601fc4dfa474a9b6c924d507d/packages/jellyfish-api-core/src/category/wallet.ts#L3-L7

Reply:
Ok sure. I just remembered we had a discussion before about enums so I used types.

Reply:
I think you convinced me not to use type for everything. I only use `type` selectively now.

Reply:
Also because of this, enum has JSDoc support but for type there isn't: https://github.com/DeFiCh/jellyfish/issues/252